apps/system/utils/Kconfig

```diff
@@ -167,7 +167,7 @@ config ENABLE_KILL
 config ENABLE_KILLALL
 	bool "killall"
 	default y
-	depends on !BUILD_PROTECTED
+	depends on FS_PROCFS && !FS_PROCFS_EXCLUDE_PROCESS
```

Comment:
Why does it have a dependency on procfs? The killall sends signal to all tasks only. It doesn't use procfs. please check.